src/grpc_server.cc

```diff
@@ -4416,12 +4416,12 @@ GRPCServer::Create(
     const std::shared_ptr<TRITONSERVER_Server>& server,
     triton::server::TraceManager* trace_manager,
     const std::shared_ptr<SharedMemoryManager>& shm_manager, int32_t port,
-    bool use_ssl, const SslOptions& ssl_options, int infer_allocation_pool_size,
-    grpc_compression_level compression_level,
+    std::string grpc_addr, bool use_ssl, const SslOptions& ssl_options,
```

Comment:
just call it `address` in grpc_server and http_server. At this level, there is no need to distinguish it.